src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -136,8 +136,13 @@ private LiKafkaConsumerImpl(LiKafkaConsumerConfig configs,
     DeliveredMessageOffsetTracker messageOffsetTracker = new DeliveredMessageOffsetTracker(maxTrackedMessagesPerPartition);
 
     // Instantiate auditor if needed.
-    Auditor<K, V> auditor = consumerAuditor != null ? consumerAuditor :
-        configs.getConfiguredInstance(LiKafkaConsumerConfig.AUDITOR_CLASS_CONFIG, Auditor.class);
+    Auditor<K, V> auditor;
```

Comment:
The previous assumption we had was that if an auditor is passed in by the users, we will use it as is. Because users has access to the auditor object and can always call `configure()` by themselves. If we call configure again it may cause configure to be called twice. Maybe we can make the documentation clearer on this?